tds-itemselection-aironline/src/main/java/tds/itemselection/algorithms/AdaptiveSelector2013.java

```diff
@@ -190,7 +190,8 @@ public ItemGroup selectNext (SQLConnection connection, List<ItemGroup> itemGroup
                     }
                 }
             }
-		    cset1 = csetFactory.MakeCset1 (connection);
+            // Allow pauses along segment barriers
+		    cset1 = csetFactory.MakeCset1 (itemGroups != null);
```

Comment:
can you explain in the comment why if the itemGroups are not null the `ignoreParent` is true?  I'm not sure I follow that and I'm guessing looking back later it might be hard to remember.